Tests/Command/CheckIpAddressIsBlacklistedCommandTest.php

```diff
@@ -0,0 +1,114 @@
+<?php
+namespace Azine\MailgunWebhooksBundle\Tests\Command;
+
+use Azine\MailgunWebhooksBundle\Command\CheckIpAddressIsBlacklistedCommand;
+use Azine\MailgunWebhooksBundle\Command\DeleteOldEntriesCommand;
+use Azine\MailgunWebhooksBundle\Services\HetrixtoolsService\HetrixtoolsServiceResponse;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\Console\Tester\CommandTester;
+use Symfony\Component\Console\Application;
+
+class CheckIpAddressIsBlacklistedCommandTest extends \PHPUnit_Framework_TestCase
+{
+    private $entityManager;
+
+    private $hetrixtoolsService;
+
+    private $azineMailgunService;
+
+    private $hetrixtoolsRespose;
+
+    public function setUp()
+    {
+        $this->hetrixtoolsRespose = new HetrixtoolsServiceResponse();
+        $this->hetrixtoolsRespose->status = HetrixtoolsServiceResponse::RESPONSE_STATUS_SUCCESS;
+        $this->hetrixtoolsRespose->api_calls_left = 5;
+        $this->hetrixtoolsRespose->blacklist_check_credits_left = 5;
+        $this->hetrixtoolsRespose->blacklisted_count = 5;
+        $this->hetrixtoolsRespose->blacklisted_on = [
+
+            [
+                'rbl' => 'dnsbl.cobion.com',
+                'delist' => 'https://exchange.test.ibmcloud.com/ip/44.44.44.44'
+            ],
+            [
+                'rbl' => 'pbl.spamhaus.org',
+                'delist' => 'https://www.test.org/query/ip/44.44.44.44'
+            ]
+        ];
+
+        $this->hetrixtoolsRespose->links = [
+            'report_link' => 'https://test.com/report/blacklist/token/',
```

Comment:
test.com is a real domain. I guess they're not happy about us using their domain. ;-) example.com is designed for this kind of things.